idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/extension/MutableStateFlow.kt

```diff
@@ -3,6 +3,15 @@ package io.github.composegears.valkyrie.ui.extension
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.update
 
-fun <T> MutableStateFlow<T>.updateState(reduce: T.() -> T) {
-    update(reduce)
+/**
+ * This is a copy of [MutableStateFlow.update] but using `this` [T] as receiver in [function].
+ */
+inline fun <T> MutableStateFlow<T>.updateState(function: T.() -> T) {
```

Comment:
I don't understand this change. `update` method do the same under the hood. 
One difference is `inline` can be added into previous function

Reply:
Yeah, we can also inline this `update` instead of copy-paste. But what, I'd prefer using `update` directly over `updateState`.

Reply:
> But what, I'd prefer using update directly over updateState.

Didn't get your point

Reply:
I mean you added `updateState` here for using `this` as the receiver, but I'd still suggest calling `update` instead of `updateState`. Either way, that's a personal point.